securedrop/journalist_templates/profile.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% block body %}
+<h2>Profile for user {{ user.username }}</h2>
+
+
+
+{% if g.user.is_admin and user.username != g.user.username: %}
+<a href="{{ url_for('admin_edit_profile', user_id=user.id) }}" id="admin-edit-profile">
+  <button type="submit" class="center">Edit This Profile</button>
+</a>
+{% elif user.username == g.user.username: %}
+<a href="{{ url_for('edit_profile') }}" id="edit-profile">
+  <button type="submit" class="center">Edit My Profile</button>
+</a>
+{% endif %}
+
+<p><b>Name:</b>
+{%  if user.full_name: %}
+{{ user.full_name }}
+{% else %}
+Not provided
+{% endif %}</p>
+
+<p><b>About this user:</b> {% if user.about: %}
+{{ user.about }}
+{% else %}
+Not provided
+{% endif %}</p>
+
+<p><b>Sources assigned to this journalist:</b></p>
+{% if sources %}
+  {% for source in sources %}
+  <p>
+    <div class="designation">
+      <span class="code-name"><a href="/col/{{ source.filesystem_id }}" id="starred-source-link-{{loop.index}}">{{ source.journalist_designation }}</a></span>
+    </div>
+  </p>
+  {% endfor %}
+{% else %}
+<p>User is not currently assigned to any sources.</p>
+{% endif %}
+
+<p><b>PGP public key:</b></p>
+{% if user.pgp_key: %}
+<pre>{{ user.pgp_key }}</pre>
```

Comment:
Maybe we don't want their entire PGP public key to be displayed here. I think a fingerprint would look nicer. We can have a "Download public key" link next to it, but journos shouldn't need each other's public keys on their JWs, so what's the point anyway?